wondrous-app/components/OnboardingDao/CreateDao/component.tsx

```diff
@@ -0,0 +1,51 @@
+import { ComponentFieldWrapper, Error, FieldInput, FieldLabel, FieldWrapper } from 'components/OnboardingDao/styles';
+import { useField } from 'formik';
+import { DescriptionCharacterLength, FieldInputDao, InputWrapper } from './styles';
+
+const Name = ({ label, ...props }) => {
+  const [field, meta] = useField(props.name);
+  return (
+    <FieldWrapper>
+      <FieldLabel>{label}</FieldLabel>
+      <FieldInput {...field} {...props} />
+      {meta.touched && meta.error && <Error>{meta.error}</Error>}
+    </FieldWrapper>
+  );
+};
+
+const Description = ({ label, maxLength, ...props }) => {
+  const [field, meta, helpers] = useField(props.name);
+  const fieldValueLength = field.value?.length ?? 0;
+  return (
+    <FieldWrapper>
+      <FieldLabel>{label}</FieldLabel>
+      <InputWrapper>
+        <FieldInputDao
+          {...field}
+          {...props}
+          onChange={(e) => {
+            const value = e.target.value;
```

Comment:
can we set this as a separate function? we don't pass additional parameters and having this as a separate func instead of inline will help maintaining it 

e.g `const handleFieldInput = (e) => {...}`